eachdist.ini

```diff
@@ -41,6 +41,12 @@ packages=
     opentelemetry-semantic-conventions
     opentelemetry-test-utils
     opentelemetry-instrumentation
+    opentelemetry-distro
+
+[exclude_release]
+packages=
+    opentelemetry-sdk-extension-aws
+    opentelemetry-propagator-aws-xray
```

Comment:
Just curious, don't we already guard ourselves from releasing these packages with this code?

https://github.com/open-telemetry/opentelemetry-python-contrib/blob/3ceab3b51071e50ef74a255b973a73f204c08c8f/scripts/build.sh#L33-L41

Reply:
This PR fixes the preparation step which bumps package versions and runs locally before the release PR is created.